guides/security/security_clearance_guidance.md

```diff
@@ -1,15 +1,46 @@
 # Guidance for Security Cleared Staff
 
-In this section we have provided some additional information for team members who are undertaking, or intend to undertake the higher level background checks for security clearance.
+## General information
+There are 3 routes for requesting Security Clearance (SC) and the route taken depends on the project.
+- Home Office - for Home Office projects only
+- MoJ - for MOJ projects only
+- Warwickshire Police - this is currently the default route for all SC and transfers
 
-## Security Clearance Guidance
+All new joiners are asked whether they already have SC and if so a process is started to transfer that clearance to Made Tech.
 
-Certain changes in personal circumstances require you to notify the government. This applies if you: 
- - Marry / form a Civil Partnership / divorce / dissolve a civil partnership or have a new or change of partner and where you have a change of co-resident (e.g. housemate).
- - Receive a Police caution, are arrested, refused bail, or convicted of an offence (other than minor road traffic offences).
- - Are subject to any type of insolvency proceedings; court repayment plan; or experience financial difficulties.
- 
-In these circumstances, you must inform the Head of People who will assist you to notify the relevant authority.
+### Timings
+Via the Warwickshire Police route we pay extra for the premium service which fast tracks the application in 25 days however that is 25 days from the point the application and all relevant information has been received. That requires the individual being cleared to fill out the forms and provide all supporting information when asked.
```

Comment:
again, feels like maybe this shouldn't be public - feel less strongly about this than disclosing the routes available to us though

Reply:
Updated